package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@salesforce/commerce",
-    "version": "248.1.5",
```

Comment:
Normally this version changes after every commit, but if you don't want to touch the version, I am fine with it. I can update this when I release it.